plugins/CoreAdminHome/OptOutManager.php

```diff
@@ -158,10 +181,333 @@ public function getQueryParameters()
     }
 
     /**
+     * Return the HTML code to be added to pages for the JavaScript opt-out
+     *
+     * @param string $piwikUrl
+     * @param string $language
+     * @param string $backgroundColor
+     * @param string $fontColor
+     * @param string $fontSize
+     * @param string $fontFamily
+     * @param bool $showIntro
+     *
+     * @return string
+     */
+    public function getOptOutJSEmbedCode(string $piwikUrl, string $language, string $backgroundColor, string $fontColor,
+                                         string $fontSize, string $fontFamily, bool $showIntro): string
+    {
+        return '<div id="matomo-opt-out"></div>
+<script src="'.$piwikUrl.'/index.php?module=CoreAdminHome&action=optOutJS&div=matomo-opt-out&language='.$language.'&backgroundColor='.$backgroundColor.'&fontColor='.$fontColor.'&fontSize='.$fontSize.'&fontFamily='.$fontFamily.'&showIntro='.($showIntro ? '1' : '0').'"></script>';
```

Comment:
Here you could apply a `rtrim($piwikUrl, '/')` to ensure the url only contains one `/`